.github/workflows/build-test.yml

```diff
@@ -46,10 +46,23 @@ jobs:
             echo "Working tree dirty at end of job"
             exit 1
           fi
+  check-workflows:
+    name: Check workflows
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download actionlint
```

Comment:
> What do you think about including these steps in the job above so that we don't have to check the repo out again?

I think it's a good time to start parallelising different steps, like we're doing in MetaMask/controllers#941. Will try to use a similar approach for this!

> If we want to keep this separate: I notice you bumped the version of actions/checkout in this job. Do we want to also change the version of actions/checkout above?

Probably a good idea, since `v2` is deprecated, AFAIK.

Reply:
I just realised that we'll need to check out the repo again regardless if we want to parallelise it. Linting the workflows doesn't require a Yarn install either, so there's no need to change it.

Will still bump the `actions/checkout` version though!